### ISSUE TITLE
Backward compat breaking change from 1.4 to 1.5

### ISSUE THREAD

**Issue by gmazzap** [composer/composer#6419]
I have a Composer plugin with a class that extends Composer `Unzipper` class.

In my plugin class I access `parent::$hasSystemUnzip` that has been `protected` since it was first introduced (in `1.0.0-beta1`, see https://github.com/composer/composer/blob/1.0.0-beta1/src/Composer/Downloader/ZipDownloader.php#L32).

In `1.4.1` (last stable at moment I'm writing) it is still protected (see https://github.com/composer/composer/blob/1.4.1/src/Composer/Downloader/ZipDownloader.php#L33).

This morning my plugin broke when tested against master because that property is now private (see https://github.com/composer/composer/blob/master/src/Composer/Downloader/ZipDownloader.php#L32).

I'm 100% in favor to private properties, but considering the class is not `final` changing a `protected` property to `private` is a breaking change, that should not be done before version 2, imo.

If you wonder, the change was introduced in 61511cf, wich should be (according to commit message) a "CS Fix" for 0a337e7c where the `$hasSystemUnzip` was cahnged from `protected` to `public`.

**Comment by Seldaek** [comment]
Yeah sorry I didn't realize one of those public properties existed previously as a protected one. Will restore it to be protected. Good thing you check things on master for compatibility.
